src/validations.js

```diff
@@ -0,0 +1,11 @@
+export function validateEmail(email) {
+  if (!email) {
+    return 'validation_messages.error_required_field';
```

Comment:
I would move this validation outside of _validateEmail_, in my opinion, it is not related. 

Imagine a scenario where the email field is optional.